utils/cliutils/commandsflags.go

```diff
@@ -1327,6 +1337,18 @@ var flagsMap = map[string]cli.Flag{
 		Name:   "format",
 		Hidden: true,
 	},
+	Filestore: cli.BoolFlag{
+		Name:  Filestore,
+		Usage: "[Default: false] Set to true to make the transfer mechanism check for the existence of artifacts in the filestore. Used when the database was loaded separately and was not indexed.` `",
```

Comment:
...artifacts in the filestore... --> ...artifacts on the target filestore...
...Used when the database was loaded separately and was not indexed --> ...Used when the files are already expected to be located on the filestore,